app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/claims/select_who_is_making_the_claim.scala.html

```diff
@@ -19,68 +19,67 @@
- pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
- paragraph: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block,
- errorSummary: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.error_summary
+    layout: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.Layout,
+    submitButton: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.button,
+    formWithCSRF: uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF,
+    dropdownDetails: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.dropdown_details,
+    radios: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.input_radio,
+    pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
+    paragraph: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.paragraph_block,
+    errorSummary: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.error_summary
 )
 
-@(form: Form[DeclarantTypeAnswer], isAmend : Boolean, router:ReimbursementRoutes)(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig)
+@(form: Form[DeclarantTypeAnswer])(implicit request: RequestWithSessionData[_], messages: Messages, viewConfig: ViewConfig, journey: JourneyBindable)
 
-    @key = @{"select-who-is-making-the-claim"}
-    @title = @{messages(s"$key.title")}
+@key = @{"select-who-is-making-the-claim"}
+@title = @{messages(s"$key.title")}
 
-    @postAction = @{ router.submitUrlForWhoIsMakingTheClaim(isAmend) }
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-    @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+@detailsParagraphs = {
+    @paragraph(Html(messages(s"$key.details.paragraph.0")))
+    @paragraph(Html(messages(s"$key.details.paragraph.1")))
+    @paragraph(Html(messages(s"$key.details.paragraph.2")))
+}
 
-    @detailsParagraphs = {
-        @paragraph(Html(messages(s"$key.details.paragraph.0")))
-        @paragraph(Html(messages(s"$key.details.paragraph.1")))
-        @paragraph(Html(messages(s"$key.details.paragraph.2")))
-    }
-
-    @layout(pageTitle = Some(s"$title"), hasErrors = hasErrors) {
+@layout(pageTitle = Some(s"$title"), hasErrors = hasErrors) {
 
-        @errorSummary(form.errors)
+    @errorSummary(form.errors)
 
-        @pageHeading(title)
+    @pageHeading(title)
 
-        @paragraph(Html(messages(s"$key.help-text")), Some("govuk-body govuk-!-margin-bottom-8"))
+    @paragraph(Html(messages(s"$key.help-text")), Some("govuk-body govuk-!-margin-bottom-8"))
 
-        @dropdownDetails(summary = s"$key.details.heading", content = detailsParagraphs)
+    @dropdownDetails(summary = s"$key.details.heading", content = detailsParagraphs)
 
-        @formWithCSRF(postAction, 'novalidate -> "novalidate") {
+    @formWithCSRF(claimRoutes.SelectWhoIsMakingTheClaimController.selectDeclarantTypeSubmit(journey), 'novalidate -> "novalidate") {
```

Comment:
I would suggest making this a parameter of the template, this will open up reuse of the template in the future if needed